File: recyclerview_album/src/main/java/com/lt/recyclerview_album/adapter/AlbumDetailAdapter.java
package com.lt.recyclerview_album.adapter;

import android.widget.ImageView;
import android.widget.TextView;

import com.lt.recyclerview_album.R;
import com.lt.recyclerview_album.base.BaseRecyclerViewAdapter;
import com.lt.recyclerview_album.base.BaseViewHolder;
import com.lt.recyclerview_album.bean.AlbumBean;
import com.lt.recyclerview_album.util.GlideUtil;

import java.util.List;

public class AlbumDetailAdapter extends BaseRecyclerViewAdapter<AlbumBean, BaseViewHolder> {

    public AlbumDetailAdapter(int layoutResId, List<AlbumBean> DataSourceList) {
        super(layoutResId, DataSourceList);
    }

    @Override
    protected void bindDataToView(AlbumBean dataSource, BaseViewHolder viewHolder, int position) {
        ImageView imageView = viewHolder.findViewById(R.id.iv_album_detail_item_img);
        GlideUtil.loadImage(viewHolder.itemView.getContext(), dataSource.getImage(), imageView);
        TextView textView = viewHolder.findViewById(R.id.tv_album_detail_item_txt);
        textView.setText(dataSource.getTitle());
    }
}

File: recyclerview_album_kotlin/src/main/java/com/lt/recyclerview_album_kotlin/adapter/AlbumDetailAdapter.kt
package com.lt.recyclerview_album_kotlin.adapter

import com.lt.recyclerview_album_kotlin.base.BaseRecyclerViewAdapter
import com.lt.recyclerview_album_kotlin.base.BaseViewHolder
import com.lt.recyclerview_album_kotlin.bean.AlbumBean
import com.lt.recyclerview_album_kotlin.util.GlideUtil
import kotlinx.android.synthetic.main.album_detail_item.view.*

class AlbumDetailAdapter(layoutResId: Int, DataSourceList: MutableList<AlbumBean>?) : BaseRecyclerViewAdapter<AlbumBean, BaseViewHolder>(layoutResId, DataSourceList) {
    override fun bindDataToView(dataSource: AlbumBean, viewHolder: BaseViewHolder, position: Int) {
        val imageView = viewHolder.itemView.iv_album_detail_item_img
        GlideUtil.loadImage(viewHolder.itemView.context, dataSource.image, imageView)
        val textView = viewHolder.itemView.tv_album_detail_item_txt
        textView.text = dataSource.title
    }
}